extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -188,7 +189,7 @@ static public function db( $db ) {
 	 */
 	static public function getDomains( $city_id, $master = false ) {
 
-		if ( ! self::isUsed() ) {
+		if ( ! static::isUsed() ) {
 			WikiaLogger::instance()->error(
 				"WikiFactory is not used.",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "WikiFactory is not used." 25 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)